### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by gautamxyz**
Completed multipliers experiment with both demo and practice, also completed posttest, prettest, theory, procedure

### CODE REVIEW

**Review by shreyash-x — APPROVED** [review]
Everything seems good

File: experiment/procedure.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
-1. With the help of Theory Page , try to build the Multiplier Circuit.
-2. If you are unable to make it , load or export the circuit component directly and use it.
-3. Then combine these component according to theory page in order to make the multiplier circuit.
-4. Then test the circuit with various input.
-5. There is an import feature, which lets you construct building blocks. Click on import and then click on circuit board to see the imported block. For more details, refer import section in manual.
+Learn everything from the 'Demo' section.
 
+In the 'Practice' section you need to make the circuit for the 2 bit multiplier. The circuit is to be made using logic gates provided, you can test your circuit after entering an input binary string and then click on simulate.
```
Comment (shreyash-x) [inline experiment/procedure.md:3]: Fix the "entering an input binary string" line.

**Review by shreyash-x — CHANGES_REQUESTED** [review]

File: experiment/procedure.md (hunk 1 above)
Comment (gautamxyz) [reply experiment/procedure.md]: done

**Comment by gautamxyz** [comment]
completed multipliers experiment

**Pull request merged**